dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/LShimMessages.nlsprops

```diff
@@ -1244,6 +1244,9 @@ CONFLICT_JMS_DESTINATION_REFERENCE_BETWEEN_WEB_FRAGMENT_XML=SRVE9953E: Two jms-d
 CONFLICT_JMS_DESTINATION_REFERENCE_BETWEEN_WEB_FRAGMENT_XML.explanation=The jms-destination of the same name cannot be configured in different web-fragment.xml files if it is missing in the web.xml.
 CONFLICT_JMS_DESTINATION_REFERENCE_BETWEEN_WEB_FRAGMENT_XML.useraction=Configure the jms-destination with the same name in the web.xml to override the ones from web-fragment.xml files.
 
+SRVE9950_WEB_FRAGMENT_XML_RESOURCE_CONFLICT=SRVE9950E: Conflicting {0} configurations with the same name {1} are found in the web-fragment.xml files of {2} and {3}.
```

Comment:
Can we keep it consistent with other messages format here?
CONFLICT_CONTEXT_SERVICE_REFERENCE_BETWEEN_WEB_FRAGMENT_XML
?

Reply:
Yes, I'll change that. My own team includes the key in the message id to make searching the code easier, but looking around in the file, it looks like this component has the opposite convention so I should do it that way instead.